app/services/travel_to_work_area_and_london_borough_service.rb

```diff
@@ -1,20 +1,62 @@
 class TravelToWorkAreaAndLondonBoroughService
   def self.add_travel_to_work_area_and_london_borough(site:)
-    url = URI("#{Settings.mapit_url}/point/4326/#{site.longitude},#{site.latitude}?type=TTW,LBO&api_key=#{Settings.mapit_api_key}")
+    new(site).add_travel_to_work_area_and_london_borough
+  rescue StandardError
```

Comment:
Can this be more specific or report the error? Not so much of an issue if this is only for the a one off backfill but if it's going to end up being used more widely than that this is going to swallow all errors.

Reply:
Yeah sorry I did not pay attention here. 
I think we actually want Sentry to deal with any issues right? 
I've removed the `rescue` and added some additional checks to ensure we only update a site upon receiving a `200` status
See https://github.com/DFE-Digital/teacher-training-api/commit/514c972f1e933e879aeb6cbab174e23cc4434b25